medical_patient_occupation/models/medical_patient_occupation.py

```diff
@@ -21,17 +21,15 @@
 #
 ###############################################################################
 
-from openerp.osv import fields, orm
+from openerp import fields, models
 
 
-class MedicalOccupation(orm.Model):
-    _name = 'medical.occupation'
+class MedicalPatientOccupation(models.Model):
+    _name = 'medical.patient.occupation'
+
+    code = fields.Char(size=256, )
+    name = fields.Char(size=256, required=True, translate=True, )
 
-    _columns = {
-        'code': fields.char(size=256, string='Code'),
-        'name': fields.char(size=256, string='Name', required=True,
-                            translate=True),
-    }
     _sql_constraints = [
         ('name_uniq', 'UNIQUE(name)', 'Name must be unique!'),
```

Comment:
I see you add a sql_constraint on code for this one: https://github.com/OCA/vertical-medical/pull/58/files Is there a reason to not doing the same here?